doc/design.rst

```diff
@@ -606,3 +597,61 @@ are deleted, the particular snapshot vanished and all snapshots
 depending on data that has been added in the snapshot cannot be restored
 completely. Restic is not designed to detect this attack.
 
+With the aforementioned assumptions and guarantees in mind, the following  are
+examples of things an adversary could achieve in various circumstances.
+
+An adversary with access to your backup storage location could: 
+- Delete or manipulate your backups, thereby impairing your ability to restore
+files from the compromised storage location. - Attempt a brute force password 
+guessing attack against a copy of the  repository (even more reason to use 
+long, random passwords).
+
+An adversary with write access to your files at the storage location could: 
+- Delete specific files (by using timestmaps to determine which files belong to
+what snapshots), resulting in snapshots vanishing and all dependent snapshots
+from being completely restored.
+
+An adversary who can passively monitor all Internet traffic could: 
```

Comment:
I'd suggest to move this section down to the other "network access" section.

Reply:
I agree it could make sense to put these two sections near each other. In my mind, "an adversary with network access" is slightly different than one "who can passively monitor all Internet traffic." However, I can see arguments either way and think having only one bullet point under the general network access odd so I ended up combining them. I debated working in "passively monitor" into the two additional points, but decided against it. If you think it's needed, let me know.